packages/beeq/src/components/date-picker/scss/bq-date-picker.scss

```
@@ -0,0 +1,240 @@
+
+.bq-date-picker__control--clear::part(button) {
+  // Since the clear button is inside the input group control,
+  // we need to reset the focus ring styles
+  --bq-ring-width: initial;
+  --bq-ring-offset-width: initial;
+  --bq-ring-color-focus: initial;
+
+  @apply h-auto rounded-xs border-none p-0;
+}
+
+/* -------------------------------------------------------------------------- */
+/*                              Prefix and suffix                             */
+/* -------------------------------------------------------------------------- */
+
+.bq-date-picker__control--prefix,
+.bq-date-picker__control--suffix {
+  @apply pointer-events-none flex items-center text-[color:var(--bq-date-picker--text-color)];
+}
+
+.bq-date-picker__control--prefix {
+  @apply me-[--bq-date-picker--gap];
+}
+
+.bq-date-picker__control--suffix {
+  @apply ms-[--bq-date-picker--gap];
+}
+
+/* -------------------------------------------------------------------------- */
+/*                         Slotted and internal icons                         */
+/* -------------------------------------------------------------------------- */
+
+bq-icon,
+::slotted(bq-icon) {
+  --bq-icon--size: var(--bq-date-picker--icon-size) !important;
+}
+
+/* -------------------------------------------------------------------------- */
+/*                         Customize Cally date picker                         */
+/* -------------------------------------------------------------------------- */
+
+calendar-date,
+calendar-range,
+calendar-multi {
+  &::part(header) {
+    @apply justify-center gap-[--bq-spacing-xs];
+  }
+
+  &::part(heading) {
+    @apply px-[--bq-spacing-xs] py-[--bq-spacing-xs2];
+  }
+
+  &::part(button) {
+    @apply border-0 p-[--bq-spacing-xs];
+
+    background-color: theme('colors.transparent');
+  }
+
+  &::part(button):focus-visible {
+    @apply rounded-s bg-[--bq-date-picker--background-color] text-[color:--bq-date-picker--border-color-focus];
+    @apply focus border-[color:--bq-date-picker--border-color-focus];
+  }
+
+  &::part(container) {
+    @apply bg-[--bq-date-picker--background-color] px-5 py-2;
+  }
+}
+
+calendar-month {
+  /* stylelint-disable-next-line custom-property-pattern */
+  --color-accent: var(--bq-stroke--brand);
+
+  &::part(button):focus-visible {
+    @apply rounded-s bg-[--bq-date-picker--background-color] text-[color:--bq-date-picker--border-color-focus];
+    @apply focus border-[color:--bq-date-picker--border-color-focus];
+  }
+
+  &::part(button) {
+    @apply h-8 w-8 rounded-[--bq-radius--s] p-1 text-[--bq-text--primary];
+
+    font-family: var(--bq-font-family);
+  }
+
+  &::part(button day selected) {
+    @apply bg-[--bq-date-picker--range-background-color] hover:bg-hover-ui-brand;
+  }
+
+  &::part(outside) {
+    @apply text-[--bq-text--secondary] opacity-100;
+  }
+
+  &::part(selected) {
+    @apply rounded-[--bq-radius--s] text-[color:var(--bq-text--inverse)];
```

Use `--bq-text--alt` for accessibility contrast

```suggestion
    @apply rounded-[--bq-radius--s] text-[color:var(--bq-text--alt)];
```